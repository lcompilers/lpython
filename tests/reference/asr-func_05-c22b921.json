{
    "basename": "asr-func_05-c22b921",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/func_05.py",
    "infile_hash": "70a0b0ddad67252c7488d322ffc565a6669e1f79b96c3ef3014b0d0a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-func_05-c22b921.stderr",
    "stderr_hash": "9611beb75ae308f86e573592164c5aca50c4e1acec2e91e48687ba96",
    "returncode": 2
}