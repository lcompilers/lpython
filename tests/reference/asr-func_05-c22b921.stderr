semantic error: Simple Type i32 cannot be intent InOut/Out
 --> tests/errors/func_05.py:3:7
  |
3 | def f(n: InOut[i32]):
  |       ^^^^^^^^^^^^^ 
